src/controllers/party.js

```diff
@@ -22,74 +23,133 @@ const createParty = (req, res) => {
       message: result.error.details[0].message,
     });
   }
-  const party = { id: parties.length + 1, ...req.body, createdOn: new Date() };
-  parties.push(party);
-  return res.status(201).json({
-    status: 201,
-    data: [party],
+
+  try {
+    const text = `INSERT INTO
+            parties(name, "hdAddress", "logoUrl")
+            VALUES($1, $2, $3)
+            returning *`;
+
+    const values = [
+      req.body.name,
+      req.body.hqAddress,
+      req.body.logoUrl,
+    ];
+
+    const { rows } = await db.query(text, values);
+
+    if (rows.length > 0) {
+      return res.status(201).json({
+        status: 201,
+        data: rows,
+      });
+    }
+  } catch (error) {
+    console.log(error);
+  }
+  return res.status(200).json({
```

Comment:
Could you check the status codes, as this one does not indicate what you are trying to pull off